vizro-core/src/vizro/models/_components/form/slider.py

```diff
@@ -43,7 +43,7 @@ class Slider(VizroBaseModel):
     min: Optional[float] = Field(None, description="Start value for slider.")
     max: Optional[float] = Field(None, description="End value for slider.")
     step: Optional[float] = Field(None, description="Step-size for marks on slider.")
-    marks: Optional[Dict[int, Union[str, Dict[str, Any]]]] = Field({}, description="Marks to be displayed on slider.")
```

Comment:
The correct typing for the value is indeed  `Dict[str, Any]` - however, we've decided to simplify it to `str` as the use case of `Dict[str, Any]` seems to be required only if one wants to style the values. This could also be achieved via CSS

Reply:
Yes, there was a discussion at the time, and it was decided to support everything that the dcc range slider supports. Styling with a dictionary is more straightforward than using custom CSS. However, you're right that it's not used often, so I'm okay with simplifying it again.

Reply:
How about the change from `int` to `float` again? Am I breaking anything with the reverting change? I tried to run the previous PR example, but I couldn't replicate the bug 🤔

Reply:
I tested it on few different examples and changing to `float` does not break anything. 👍